### ISSUE TITLE
Create jet logo/icon

### ISSUE THREAD

**Issue by lread** [borkdude/jet#50]
The wise borkdude (perhaps unwisely 🙂) asked me if I'd be interested in creating a logo/icon for jet.

I am an amateur at this (at best) but he liked my [rewrite-cljc logo ](https://github.com/lread/rewrite-cljs-playground/blob/master/doc/rewrite-cljc-logo.png)and my [adaptation of the clj-kondo logo to icon](https://github.com/borkdude/clj-kondo/blob/master/logo/icon.png).

I'm happy to give it a shot, and sincerely won't be abashed if my attempts miss the mark.

I'll share ideas and sketches in this issue.

**Comment by lread** [comment]
So, what is jet?  Current slogan is "CLI to transform between JSON, EDN and Transit, powered with a minimal query language".

So we have: transform, query, json, edn, transit.

**Sample 1 - cursive**
Transformation implies that one thing can be connected to the other, so using a cursive font could be interesting. Here is a sample using the "engagements" font I found over at google fonts:

![jet-cursive1](https://user-images.githubusercontent.com/967328/67641306-da8f6b00-f8d7-11e9-8679-9da54f686d21.png)

**Sample 2 - stylistic**
I wondered what it would look like if every letter was joined (including the j and the t) and came up with this:

![jet-stylistic](https://user-images.githubusercontent.com/967328/67641554-14ad3c80-f8d9-11e9-920e-b70be4aadc21.png)

**Sample 3 - stylistic with arrows**
I thought maybe some arrows would help indicate transformation.

![jet-stylistic-arrows](https://user-images.githubusercontent.com/967328/67641566-21ca2b80-f8d9-11e9-972a-ab0b50b0bdfa.png)

**Sample 4 - playing with characters**
I then thought it might be fun to play with characters which are common in edn/json/transit. I thought `:` was a good one for edn `{` covers json and maybe `[` covers transit.

![jet-face-screen](https://user-images.githubusercontent.com/967328/67641612-7e2d4b00-f8d9-11e9-9046-d4e044f1d80c.png)

The `..` (a colon on its side) could mean transform (or waiting for a search result, I suppose). Also the `}` looks a bit like a face and the `[` looks like a screen.

**Sample 5 - playing with characters - face**
I liked 4 and it made me think of playing further with the same characters.

![jet-face](https://user-images.githubusercontent.com/967328/67641621-a9179f00-f8d9-11e9-8615-87095f08225b.png)

**Sample 6 - playing with characters - shorter face**
I liked sample 5, but it is a bit tall for an icon, so fiddled and came up with this:

![jet-shorter-face](https://user-images.githubusercontent.com/967328/67641624-b46aca80-f8d9-11e9-8266-b9a68fc017f3.png)

So that's it for round 1, and should be enough to get some more ideas flowing.  Let me know what you think.

**Comment by borkdude** [comment]
@lread I think I prefer example 1, just the most readable and elegant one. 

Maybe we can make a pun on the relation to jet engines / airplanes and that it has fast startup?
Not sure if that works out :)

**Comment by lread** [comment]
Cool. The advantage of such a short name means that it can also work as an icon.

As for the slogan, not sure you can beat clj-kondo's "a linter that sparks joy". That is pure magic naming.

Uh...
"Supersonic transforming and querying at the command line"

Hmm...
"Your turboprops to fast transformations and queries"

Gee... this is hard...
"An engine of change"

**Comment by lread** [comment]
Oh... so you meant a visual pun.  I'll stop thinking of slogans! 🙂

**Comment by lread** [comment]
**Sample 7 - jet**

![jet-jet](https://user-images.githubusercontent.com/967328/67642810-0ade0600-f8e6-11e9-8e86-f95a622e813a.png)

**Comment by lread** [comment]
**Sample 8 - rocket**

![jet-rocket](https://user-images.githubusercontent.com/967328/67643850-1fc09680-f8f2-11e9-853e-faacb42e78b8.png)

**Comment by borkdude** [comment]
@lread Is is possible to make a mix between 7 and 8? I kind of like 7 better, but with a rocket instead of an airplane? :-)

**Comment by borkdude** [comment]
Btw, I already like 7, that's my favorite so far! Nice job!

**Comment by lread** [comment]
Before I do another rocket, here's an experiment with speed lines.

**Sample 9 - speed lines**
![jet-speed-lines](https://user-images.githubusercontent.com/967328/67698676-277b4c00-f981-11e9-8cd5-3c086d47e0b2.png)

**Comment by borkdude** [comment]
@lread I like it!

**Comment by lread** [comment]
Ya me too!

For completeness, more rockets. 

**Sample 10 - rocket revisited**

![jet-rocket-squat](https://user-images.githubusercontent.com/967328/67705970-5ea42a00-f98e-11e9-8217-fc0319367d27.png)

**Sample 11 - rocket - slimmer**

![jet-rocket-tall](https://user-images.githubusercontent.com/967328/67706141-9612d680-f98e-11e9-9d60-19087b4214a2.png)

**Comment by borkdude** [comment]
@lread Sample 10 sparks joy as well. Maybe it's fun to put the letters on the rocket as if it's printed on it? Like the Apollo 11 :-)

**Comment by borkdude** [comment]
@lread Yeah, I see it now, but it wasn't immediately apparent to me. Still I think 9 is the winner.

**Comment by lread** [comment]
**Sample 12 - faster one font**
Saw a font I did not notice over at google fonts. Looks maybe a bit cheesy? But so very very fast.

![jet-faster-one-font](https://user-images.githubusercontent.com/967328/67717375-f95b3380-f9a3-11e9-997c-a5e518542a77.png)

**Comment by borkdude** [comment]
@lread Sorry, not a fan of 12 :).

What about playing with the letters like this?

```
JET
===
SDR
ONA
N N
  S
  I
  T
```

Probably not the best idea ;-).

**Comment by lread** [comment]
Yes, #12 was included just to make the others look better. 🙂 

Interesting idea.  Could make for a nice tall rocket but maybe letters will be too small?  Probably worth a try.

**Comment by lread** [comment]
I have a new appreciation for novice graphic artists. 🙂 

**sample 13 - horizontal rocket - text enclosed**

![jet-horizontal-rocket-name-enclosed](https://user-images.githubusercontent.com/967328/67724165-86a68400-f9b4-11e9-9bf6-89222da4ee77.png)

**Comment by borkdude** [comment]
Cute, this makes me smile :). 9 is still my fav.

**Comment by lread** [comment]
The rocket with letters is tricky (for me at least!), here's a work in progress (currently looks like a graphic from the 1990s to me!). I think it is too busy and doesn't merit more work but feel free to disagree. Who knows, it might look better with more polish. Lemme know if you are interested in pursuing this one more. 

It is descriptive, I'll give you that! 

Also: it might be too specific, are you thinking of ever supporting more formats?

**sample 14 - rocket - lotsa letters - unfinished**
![jet-rocket-lotsa-letters](https://user-images.githubusercontent.com/967328/67727894-8f518700-f9c1-11e9-9178-8b7f7f537416.png)

**Comment by borkdude** [comment]
@lread you're right, it's probably too specific and also a little bit too busy.

9 is still on top for me.
